clients/nexus-iq/src/main/kotlin/NexusIqService.kt

```diff
@@ -42,6 +42,18 @@ import retrofit2.http.Path
  */
 interface NexusIqService {
     companion object {
+        /** Identifier of the scoring system _Common Vulnerability Scoring System_ version 2. */
```

Comment:
Commit message: You state a reference [1] but the corresponding URL seems to be missing.

Reply:
Ops, thanks.